### PR TITLE
Feature/7

### PR DESCRIPTION
**Pull request by pfif**
Issue #7

### CODE REVIEW

File: pyrestriction/model.py [hunk 1]
```diff
@@ -121,10 +121,13 @@ class Operation:
 
     Its method "next" is responsible to create the Operation for the next period.
     If there is no next operation, it must raise NoNextOperation.
+
+    "purpose" stores a short sentence describing why the operation exists.
     """
-    def __init__(self, amount, debt=False):
+    def __init__(self, amount, purpose, debt=False):
```
Comment (matthieuchoplin) [inline pyrestriction/model.py:127]: do you want "purpose" to be required?

File: tests/test_everything.py [hunk 2]
```diff
@@ -171,11 +171,25 @@ def test_avaliable(self):
             self._instance.avaliable()
 
 
-class SavingOperationTest(unittest.TestCase):
+class TestPuposeMixin(object):
```
Comment (matthieuchoplin) [inline tests/test_everything.py]: typo: "Pupose" -> "Purpose"

File: pyrestriction/model.py (hunk 1 above)
Comment (floostmodern) [inline pyrestriction/model.py]: Yes I do. This makes account files clearer.

Comment (pfif) [inline pyrestriction/model.py:127]: Yes, it makes account files and the output of the program clearer, I find.

File: tests/test_everything.py (hunk 2 above)
Comment (pfif) [inline tests/test_everything.py]: Thanks! I knew it was worth it being CR haha

**Comment by pfif** [comment]
rebuild

**Pull request merged**